server/testutils/factories/index.ts

```diff
@@ -6,6 +6,7 @@ import courseParticipationFactory from './courseParticipation'
 import courseParticipationOutcomeFactory from './courseParticipationOutcome'
 import coursePrerequisiteFactory from './coursePrerequisite'
 import inmateDetailFactory from './inmateDetail'
+import offenceDetailFactory from './offenceDetail'
```

Comment:
Open to suggestions on naming as is obviously similar to offenceDetail**s**Factory